$OpenBSD: patch-src_md5_c,v 1.1 2012/05/03 06:51:07 ajacoutot Exp $

Use the config.h header to set WORDS_BIGENDIAN instead of another means of checking.

--- src/md5.c.orig	Mon Apr 30 01:22:39 2012
+++ src/md5.c	Mon Apr 30 01:22:56 2012
@@ -28,12 +28,7 @@
 #include "md5.h"
 /* #include "unlocked-io.h" */
 
-#ifdef _LIBC
-# include <endian.h>
-# if __BYTE_ORDER == __BIG_ENDIAN
-#  define WORDS_BIGENDIAN 1
-# endif
-#endif
+#include "config.h"
 
 #ifdef WORDS_BIGENDIAN
 # define SWAP(n)                                                        \
